examples/gno.land/p/demo/int256/conversion.gno

```diff
@@ -79,8 +79,9 @@ func (z *Int) ToString() string {
 	}
 
 	t := z.abs.Dec()
-	if z.neg {
+	if z.neg && !z.abs.IsZero() {
```

Comment:
What is this? Defensive programming in case it missed a check somewhere to correct the sign? 😆 Do any of the tests fail if you remove this?

Reply:
i forgot to delete it 😅. thanks